modules/@apostrophecms/asset/lib/webpack.config.js

```diff
@@ -1,5 +1,5 @@
 const path = require('path');
-const merge = require('webpack-merge');
+const merge = require('webpack-merge').merge;
```

Comment:
New version of webpack-merge offers no "smart" method, suggests this way of requiring and using. Seems OK so far.